src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu__list {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using tag names for styling. Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element`, and `[attribute]`. For example, use `.menu__list` instead of styling the `ul` tag directly.